tests/testthat/test-get-dupes.R

```diff
@@ -37,3 +37,9 @@ test_that("works on variables with irregular names", {
   ) # does it return the right-sized result?
   expect_is(badname_df %>% get_dupes(), "data.frame") # test for success, i.e., produces a data.frame (with 0 rows)
 })
+
+test_that("tidyselect specification matches exact specification", {
+  expect_equal(mtcars %>% get_dupes(contains("cy"), mpg), mtcars %>% get_dupes(cyl, mpg))
+  expect_equal(mtcars %>% get_dupes(mpg), mtcars %>% get_dupes(-c(cyl, disp, hp, drat, wt, qsec, vs, am ,gear, carb)))
+  expect_equal(suppressMessages(mtcars %>% select(cyl, wt) %>% get_dupes()), mtcars %>% select(cyl, wt) %>% get_dupes(everything()))
```

Comment:
The `suppressMessages()` call suggests that there may be an issue here.  If a message or warning is desired, please use `expect_warning()` or `expect_message()` so that the test includes a test for expected behavior.  If a message or warning is not desired, please update the code so that the warning/message is not generated.

Reply:
The message from an empty `get_dupes()` call is a custom message that says `"No variable names specified - using all columns.\n"` (line 28).  This message doesn't occur when using `everything()` because you have specified variable names. My test was intended to compare the outputs of the two calls while ignoring that one had that custom message.  What's a better way to do this?

Reply:
You're good here then, thanks for explaining.